### DISCUSSION TITLE
How to fix “error recovering your device”

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#1306] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
[
![IMG_0395](https://github.com/user-attachments/assets/1dc04856-5e10-4b91-82e9-680d384f8ded)
](url)

How to fix this, before you ask me my kernver is at 0 also here other information.
Model:HP Chromebook 11 G9 EE
Code name:drawlat
Board:dedede
Gbb flags:0x8090
WP:off
Trying to go to v107

**Comment by Sam-hacks1** [comment]
yo you got the exact same model, code name, and board

**Comment by schoolexploitkid** [comment, marked as answer]
ima ask some random stuff, but did u brick anything while disabling wp? if not, then listen to the directions and try again.
